collections/_infographics/policies/emission-pledges.md

```diff
@@ -12,4 +12,34 @@ data-orig:
   - [ "Our Jupyter notebook", "https://github.com/faktaoklimatu/data-analysis/blob/master/notebooks/net-zero-pledges.ipynb" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/emisni-zavazky" %}
+## How to read an infographic?
+{% comment %}
+{% include preview-box.html
+    title="What is carbon neutrality?"
+    text="We explain exactly what the zero carbon commitment means and how the term can be confusing."
+    slug="uhlikova-neutralita"
+%}
+{% endcomment %}
+This infographic shows the share of carbon-neutral countries in global CO<sub>2</sub> emissions in 2018. Excluding the Land Use, Land Use Change and Forestry sector (*LULUCF), all CO<sub>2</sub> emissions, including international maritime and aviation transport. **As of March 2022, 87% of global emissions come from countries moving towards carbon neutrality**, with 131 countries total.
+
+Countries that are moving towards carbon neutrality are those that have commitments: fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage. If we exclude countries with commitments in the draft or discussion phase, there would be a 3% reduction in covered emissions to 84% of global emissions.
```

Comment:
slight rephrase for clarity:

Countries that are moving towards carbon neutrality are those whose commitments are either fulfilled (e.g., Bhutan), set in law, set in a policy document, in the form of a declaration/commitment, or at the proposal/discussion stage.